updateLeantime.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/bash
+
+#Declaring used colors
+BLUE='\033[1;34m'
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+NC='\033[0m'
+
+#Declaring used functions
+getConfigVar () {
+	grep "\$$1" config/configuration.php |awk -F "=" '{print $2}' |awk -F ";" '{print $1}' |tr -d ' ' |tr -d "\'"
+}
+
+getDateString () {
+	date '+%Y%m%d_%H%M%S'
+}
+
+echo -e "${BLUE}"
+echo "    _                 _   _                             "
+echo "   | |   ___ __ _ _ _| |_(_)_ __  ___                   "
+echo "   | |__/ -_) _\` | ' \  _| | '  \/ -_)                  "
+echo "   |____\___\__,_|_||_\__|_|_|_|_\___|      _           "
+echo "                     | | | |_ __  __| |__ _| |_ ___ _ _ "
+echo "                     | |_| | '_ \/ _\` / _\` |  _/ -_) '_|"
+echo "                      \___/| .__/\__,_\__,_|\__\___|_|  "
+echo "                           |_|                          "
+echo -e "${NC}"
+
+echo -e "\nFor correct operation of this script, please ensure you have zip, unzip, wget and curl installed.\n"
+
+CURRENT_VERSION=$(grep "appVersion" config/appSettings.php |awk -F' = ' '{print "v"substr($2,2,length($2)-3)}')
+LATEST_RELEASE=$(curl -L -s -H 'Accept: application/json' https://github.com/leantime/leantime/releases/latest)
+LATEST_VERSION=$(echo $LATEST_RELEASE | sed -e 's/.*"tag_name":"\([^"]*\)".*/\1/')
```

Comment:
Had some issues with this in other linux environements. Stange since it is all bash...